website/snippets/tutorial-change-way-model-materialized.md

```diff
@@ -2,7 +2,16 @@ One of the most powerful features of dbt is that you can change the way a model
 
 By default, everything gets created as a view. You can override that by materializing everything in jaffle_shop as a table.  Everything in the example project will still be materialized as a view.
 
-1. Edit your `dbt_project.yml` file:
+1. Edit your `dbt_project.yml` file.
+  - Replace the value for `name` with:
+    <File name='dbt_project.yml'>
+
+    ```yaml
+    name: 'jaffle_shop'
+    ```
+
+    </File>
+  - Replace the value for `models` with:
```

Comment:
would edit to say: 
"- Replace the the project name under `models` with: "
since it's not quite a value for models.